ivy_tests/test_ivy/helpers/test_parameter_flags.py

```diff
@@ -48,6 +48,7 @@ def _as_varaible_strategy(draw):
 BuiltFrontendArrayStrategy = DynamicFlag(st.booleans())
 BuiltTranspileStrategy = DynamicFlag(st.just(False))
 BuiltPrecisionModeStrategy = DynamicFlag(st.booleans())
+BuiltCythonWrapperStrategy = DynamicFlag(st.booleans())
```

Comment:
let's set `BuiltCythonWrapperStrategy` to be `DynamicFlag(st.just(False))` for now, we need to figure out a way to update the database with those failures separate to the ones with the python function wrappers. That's because the cython wrapper code isn't public and so the failures emerging from the cython wrappers can't be fixed by the os community

Reply:
2. Got it!
1. For some reason it does not work as expected without that line. I did verify it when you first mentioned it. 😅

Reply:
ig that might be because the mode is missing from [this](https://github.com/unifyai/ivy/pull/27770/files#diff-8f7902fadeb2dd53265eb3cfbeb649d12f543d777ebd7f85225d4431b76f067cR1450) list? I'd suggest searching for all occurrences of `array_mode` or some other mode and verifying if the equivalents are present for the `cython_wrappers_mode`, as ideally we wouldn't need to  do any specific assignment statement for a mode in the backend handler 😄

Reply:
Thanks for pointing it out @vedpatwardhan! I'd completely missed that 🤦